src/intersection-observer-polyfill.js

```diff
@@ -108,13 +55,13 @@ export class IntersectionObserverPolyfill {
      * A list of threshold, sorted in increasing numeric order
      * @private @const {!Array}
      */
-    this.threshold_ = opt_option ? opt_option.threshold : DEFAULT_THRESHOLD;
+    this.threshold_ = opt_option ? opt_option.threshold : [0];
+    this.threshold_ = this.threshold_.sort();
```

Comment:
Code smell to modify objects you don't own (or that you're not expected to modify).